### PR TITLE
Incorporate Jenkinsfile changes from #176

### PR DESCRIPTION
**Pull request by slide**

### CODE REVIEW

**Comment by batmat** [comment]
@alecharp failing on findbugs IIUC. Was it working on Java 11 during your testing? Thanks!

File: Jenkinsfile [hunk 1]
```diff
@@ -1 +1,9 @@
-buildPlugin(failFast: false, jdkVersions: [8, 11])
+buildPlugin(
+    failFast: false,
+    configurations: [
+        [ platform: 'linux', jdk: "8", jenkins: null ],
+        [ platform: 'windows', jdk: "8", jenkins: null ],
+        [ platform: 'linux', jdk: "11", jenkins: "2.163", javaLevel: "8" ],
```
Comment (oleg-nenashev) [inline Jenkinsfile]: Use 2.164 now? It is going to be the LTS baseline

```suggestion
        [ platform: 'linux', jdk: "11", jenkins: "2.164", javaLevel: "8" ],
```

File: Jenkinsfile [hunk 2]
```diff
@@ -1 +1,9 @@
...
+        [ platform: 'windows', jdk: "11", jenkins: "2.163", javaLevel: "8" ],
```
Comment (oleg-nenashev) [inline Jenkinsfile]:
```suggestion
        [ platform: 'windows', jdk: "11", jenkins: "2.164", javaLevel: "8" ],
```

**Comment by batmat** [comment]
Superseded by https://github.com/jenkinsci/email-ext-plugin/pull/183

**Pull request closed without merging**